solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-10.adoc

```diff
@@ -42,3 +42,15 @@ has been removed. Please use `-Dsolr.hiddenSysProps` or the envVar `SOLR_HIDDEN_
 * The `<hiddenSysProps>` solr.xml element under `<metrics>` has been removed. Instead use the `<hiddenSysProps>` tag under `<solr>`, which accepts a comma-separated string.
 Please see `-Dsolr.redaction.system.pattern`, which allows users to provide a pattern to match sysProps that should be redacted for sensitive information,
 has been removed. Please use `-Dsolr.hiddenSysProps` or the envVar `SOLR_HIDDEN_SYS_PROPS` instead.
+
+=== SolrCLI
+
+The Solr CLI has gone through some signficant rennovations to reduce technical debt, and function more consistently and predictably.
+You will need to review any custom scripts that use the `bin/solr` command to ensure that you are using the correct options. 
+
+Here are changes from Solr 9.3 and Solr 10 in the CLI.
```

Comment:
You can remove this line, as these changes are only from 10.0 onwards?

Reply:
Correct...   This is way to hard to make be backward compatible...     Do I need to list out the before or after in the changelog, or would it be enough to say "See the Reference Guide page XXX for the long parameter names for each command."

Reply:
I've made the change that @janhoy  suggested!